operators/nftopology/README.md

```diff
@@ -0,0 +1,152 @@
+# NFTopology Controller
+- Author: @s3wong
+- Status: Work-in-Progress
+- Approver: 
+
+## Description
+NFTopology controller (NTC) takes a network topology centric input and kickstarts the Nephio automation process via generating a PackageVariantSet (PVS) custom resource, and as the associated packages are being deployed, NTC continuously updates statuses to reflect state of deployment of these packages. Primarily, NTC serves as example application to utilize Nephio primitives for network function automations.
+
+![NFTopology High Level Functional Diagram](./img/nftopology-functional.jpg)
+
+NTC contains two controllers, one watches the NFTopology custom resource, and reconcile that intent via constructing a number of PVSs; the other controller watches PackageRevision resources in the management cluster to determine what NF package has been deployed, and updates the NFTopology status accordingly.
```

Comment:
NTC contains two controllers. The first watches the NFTopology custom resource and reconcile that intent by constructing a number of PVSs. The other controller watches PackageRevision resources in the management cluster to determine what NF packages have been deployed and updates the NFTopology status accordingly.

Reply:
Comments incorporated. Thanks